src/alt.js

```diff
@@ -454,6 +461,7 @@ function createStoreFromClass(alt, StoreModel, key, ...argsForConstructor) {
 
 class Alt {
   constructor(config = {}) {
+    this.config = config
```

Comment:
I figure I would need the entire config at some point in the future, no?. Should I put it in a symbol?